src/main/java/core/basesyntax/classes/figures/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.classes.figures;
+
+public class Square extends Figure {
+    private final int side;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.